src/scripts/install_machine.py

```diff
@@ -54,11 +54,35 @@ def get_graphics_capabilies(connection):
 
 
 def prepare_graphics_params(connection):
+    # Use ipv6 as workaround for https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
+    spice_address = '::1'
+
+    # Check to make sure loopback has an IPv6 address assigned
+    # if not default back to IPv4
+    try:
+        if subprocess.run(['ip'], shell=True).returncode == 0:
+            ipv6_loopback = subprocess.check_output(['ip', '-6', 'addr', 'show', 'dev', 'lo'])
+
+            if not "lo:" in str(ipv6_loopback):
```

Comment:
Pretty sure you mean "inet6" here? You asked for `lo` already.

Reply:
Since we've got the -6 flag, this still works as intended, as if there's no IPv6 assignment lo: wouldn't be displayed since it doesn't show the device if there's no ip assigned, that being said inet6 feels like a better bit of text to search for so I've updated that with this set of changes